### PR TITLE
Create orbisdata_plot_with_geopandas.py

### PR DESCRIPTION
**Pull request by ernlt**

### CODE REVIEW

File: orbisdata_plot_with_geopandas.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+
+# Data location
+dirfile = "/home/erenault/geoclimate/osm/output/osm_vannes/"
```
Comment (ebocher) [inline orbisdata_plot_with_geopandas.py:8]: use a generic path
/tmp/my...

File: orbisdata_plot_with_geopandas.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+
+# Data location
+dirfile = "/home/erenault/geoclimate/osm/output/osm_vannes/"  
+
+# Example of plot for a geojson variable.
+plt.rcParams['figure.figsize'] = (20, 10)
+data = gpd.read_file(dirfile+"block_indicators.geojson")
```
Comment (ebocher) [inline orbisdata_plot_with_geopandas.py:12]: Why using a dir ? what about a file path

**Comment by ebocher** [comment]
By the way, thanks for this first contrib

File: orbisdata_plot_with_geopandas.py (hunk 2 above)
Comment (ernlt) [reply orbisdata_plot_with_geopandas.py:12]: Yes, one can use the **os** module with **import os**.
Given the generic path, for example 'generic_path = /tmp/data/', line 12 becomes:
data = gpd.read_file(os.path.join(generic_path, "block_indicators.geojson")

File: orbisdata_plot_with_geopandas.py (hunk 1 above)
Comment (ebocher) [inline orbisdata_plot_with_geopandas.py:8]: Waiting for a fix

**Pull request closed without merging**